src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandResult.java

```diff
@@ -168,7 +168,10 @@ public List<JsonNode> getResponseDocuments() {
             implementation = String.class)
       })
   public record Error(
-      String message, @JsonAnyGetter @Schema(hidden = true) Map<String, Object> fields) {
+      String message,
+      @JsonAnyGetter @Schema(hidden = true) Map<String, Object> fields,
+      // Http status code to be used in the response, defaulted to 200
+      @JsonIgnore int errorCode) {
```

Comment:
I would rather accept something that is not int, but HttpStatus code object from JAX-RS.. Now I can set status to `-522` and would fail of course at some point later on.. Since we are anyway ignoring the field, I guess that's better option imo..

Reply:
@ivansenic Make sense. Not sure if we should be using JAX-RS enum everywhere in the code. Better to create separate enum for the error code and later map it when result is generated?

Reply:
I would use the JAX-RS directlym this is anyway API level code.. Having our own is really an overhead imo..

Reply:
Thought of not using JAX-RS enum in classes like ThrowableToErrorMapper.